src/selectors/bridgeSelector.ts

```diff
@@ -35,3 +35,6 @@ export const selectIsFetchTxBlockNumber = () => (state) =>
 
 export const selectIsTeleportationOfAssetSupported = () => (state) =>
   state.bridge.isTeleportationOfAssetSupported
+
+export const selectReenterWithdrawal = () => (state) =>
```

Comment:
```suggestion
export const selectReenterWithdrawalConfig = () => (state) =>
```

This will be more clear the purpose of selector